src/extensions/mountainview/MVSortingView/MVSortingView.tsx

```diff
@@ -73,44 +74,32 @@ export const openViewsReducer: React.Reducer<View[], OpenViewsAction> = (state:
 }
 
 const MVSortingView: FunctionComponent<SortingViewProps> = (props) => {
+    // useCheckForChanges('MVSortingView', props)
     const {plugins, recording} = props
     const [openViews, openViewsDispatch] = useReducer(openViewsReducer, [])
     const unitsTablePlugin = plugins.sortingViews.UnitsTable
     const averageWaveformsPlugin = plugins.sortingViews.AverageWaveforms
-    const electrodeGeometryPlugin = plugins.sortingViews.ElectrodeGeometrySortingView
-    const [initialized, setInitialized] = useState(false)
-    useEffect(() => {
-        if ((openViews.length === 0) && (!initialized)) {
-            setInitialized(true)
-            // if (electrodeGeometryPlugin) {
-            //     openViewsDispatch({
-            //         type: 'AddView',
-            //         plugin: electrodeGeometryPlugin,
-            //         pluginType: 'SortingUnitView',
-            //         label: electrodeGeometryPlugin.label,
-            //         area: 'north'
-            //     })
-            // }
-            if (unitsTablePlugin) {
-                openViewsDispatch({
-                    type: 'AddView',
-                    plugin: unitsTablePlugin,
-                    pluginType: 'SortingUnitView',
-                    label: unitsTablePlugin.label,
-                    area: 'north'
-                })
-            }
-            if (averageWaveformsPlugin) {
-                openViewsDispatch({
-                    type: 'AddView',
-                    plugin: averageWaveformsPlugin,
-                    pluginType: 'SortingUnitView',
-                    label: averageWaveformsPlugin.label,
-                    area: 'south'
-                })
-            }
+    // const electrodeGeometryPlugin = plugins.sortingViews.ElectrodeGeometrySortingView
+    useOnce(() => {
```

Comment:
The useOnce hook is useful here so we don't need to have initialized state, etc.